krkn/scenario_plugins/node_actions/gcp_node_scenarios.py

```diff
@@ -14,13 +14,15 @@
 
 class GCP:
     def __init__(self):
-        try:
+        if "GOOGLE_APPLICATION_CREDENTIALS" in os.environ:
             gapp_creds = os.getenv("GOOGLE_APPLICATION_CREDENTIALS")
             with open(gapp_creds, "r") as f:
                 f_str = f.read()
                 self.project = json.loads(f_str)["project_id"]
             # self.project = runcommand.invoke("gcloud config get-value project").split("/n")[0].strip()
```

Comment:
Why do we keep this commented line?

Reply:
Feel free to cherry-pick only the feature commit if desired.